package.json

```diff
@@ -46,11 +51,6 @@
     "protractor": "~5.4.3",
     "ts-node": "~8.3.0",
     "tslint": "~5.18.0",
-    "typescript": "~3.7.5",
-    "@angular-devkit/architect": "^0.900.0-0 || ^0.900.0",
-    "firebase-tools": "^7.12.1",
-    "fuzzy": "^0.1.3",
-    "inquirer": "^6.2.2",
-    "inquirer-autocomplete-prompt": "^1.0.1"
```

Comment:
Any reason why this was removed?

Reply:
A lot of them were moved up or down. None of them were actually removed!